### PR TITLE
Sample HPCM cluster configuration

### PR DESCRIPTION
**Pull request by nbwhite**
Updates to the aurora configuration to work with a sample HPCM cluster

### CODE REVIEW

**Review by sindhu-106 — APPROVED** [review]
Nearline, volt, Provisioner adapter changes looks good to me.

**Review by avincigu — APPROVED** [review]

File: configurations/aurora/ProviderMonitoringNetworkForeignBus.json [hunk 1]
```diff
@@ -1,69 +1,20 @@
...
-        "tokenServer": "https://api-gw-service-nmn.local/keycloak/",
-        "clientId": "admin-client",
-        "clientSecret": "dd0f8dee-f40b-4b16-a5dd-af2ed1b8a99c",
-        "realm": "shasta"
+        "bootstrap.servers": "admin:9092",
+	"group.id": "dai-monitoring",
+	"schema.registry.url": "http://admin:8081",
+	"value.deserializer": "io.confluent.kafka.serializers.KafkaAvroDeserializer",
+	"auto.commit.enable": true,
+        "topics": "sensors_node"
```
Comment (paul-amonson) [inline configurations/aurora/ProviderMonitoringNetworkForeignBus.json:15]: Indentation inconsistent.

File: configurations/aurora/ProviderProvisionerNetworkForeignBus.json [hunk 2]
```diff
@@ -6,16 +6,14 @@
...
-        "clientId": "admin-client",
-        "clientSecret": "dd0f8dee-f40b-4b16-a5dd-af2ed1b8a99c",
-        "realm": "shasta"
+	"bootstrap.servers": "admin:9092",
+        "group.id": "boot_states",
+        "schema.registry.url": "http://admin:8081",
+        "auto.commit.enable": false,
+	"auto.offset.reset": "earliest",
+        "topics": "SYSLOG,HPCMLOG"
       },
```
Comment (paul-amonson) [inline configurations/aurora/ProviderProvisionerNetworkForeignBus.json:15]: Indentation inconsistent.

**Review by paul-amonson — APPROVED** [review]
Minor comments but I won't block the review for this.

**Pull request merged**